src/core/components/layout/components/inline/styles.ts

```diff
@@ -0,0 +1,41 @@
+import { css, SerializedStyles } from "@emotion/core"
+import { space } from "@guardian/src-foundations"
+import { InlineSpace } from "./inline"
+
+export const inline = css`
+	display: flex;
+	flex-wrap: wrap;
+`
+
+const inlineSpaceStyle = (number: InlineSpace) => css`
+	& > * {
+		margin-left: ${space[number]}px;
+		margin-bottom: ${space[number]}px;
+	}
+`
+
+export const removeLeftMargin = (number: InlineSpace) => css`
+	margin: 0 -${space[number]}px;
```

Comment:
This style gets applied to the same element as the styles produced by `inlineSpaceStyle`. It even accepts the same argument. Perhaps we can move the negative margin up to `inlineSpaceStyle`?